en/asset/cache-manager.md

```diff
@@ -45,29 +71,35 @@ assetManager.loadRemote('http://example.com/background.jpg', function (err, text
 });
 ```
 
-## Cache resources
+## Caching resources
 
-A number of parameters are provided in the Cache Manager to control the caching of resources.
+A number of parameters are provided in the cache manager to control the caching of resources:
 
-1. `cacheManager.cacheDir` -- Controls the storage directory for cached resources.
-2. `cacheManager.cacheInterval` -- Controls the period of caching a single resource, the default is once every 500ms.
-3. `cacheManager.cacheEnabled` -- Controls whether or not to cache resources, which defaults to caching. Alternatively, you can override the global settings by specifying the optional parameter `cacheEnabled`, for example:
+- `cacheManager.cacheDir` -- controls the directory where the cached resources are stored.
+- `cacheManager.cacheInterval` -- controls the period of caching a single resource, default 500 ms once.
+- `cacheManager.cacheEnabled` -- controls whether to cache the resource, default is cache. Alternatively, developers can override the global setting by specifying the optional parameter ``cacheEnabled``, e.g.
 
   ```typescript
   assetManager.loadRemote('http://example.com/background.jpg', {cacheEnabled: true}, callback);
   ```
 
-## Clear cache
+### Clearing cache resources
 
-The cache manager provides three interfaces `removeCache`, `clearCache` and `clearLRU` to clean up cache resources.
+If the cache resource exceeds the limit and the developer needs to clear the resource manually, use `removeCache`, `clearCache`, `clearLRU` provided by the cache manager `cc.assetManager.cacheManager` to clear the cache resource.
 
-- `removeCache` -- Clean up a single cached resource, and you need to provide the original path of the resource when using it.
+- `clearCache` -- clears all cache resources in the cache directory, please use with caution.
+- `clearLRU` -- clears the older resources in the cache directory. The mini-game platform will automatically call `clearLRU` when the cache space is full.
+- ``removeCache`` -- clears a single cache resource. To use it, provide the original path of the resource, e.g.
 
   ```typescript
   assetManager.loadRemote('http://example.com/background.jpg', function (err, texture) {
       assetManager.cacheManager.removeCache(texture.nativeUrl);
   });
   ```
 
-- `clearCache` -- Clean up all cache resources, please use it carefully.
-- `clearLRU` -- Clean up older resources. The mini game platform will automatically call `clearLRU` when the cache space is full.
+When the developer upgrades the engine version, the cache resources left locally are still the same as those corresponding to the previous old version of the engine and are not automatically cleared. This may lead to problems such as errors in loading resources or rendering errors. There are two solutions.
```

Comment:
```suggestion
When the developer upgrades the engine version, the cached resources left locally are still the resources corresponding to the previous old version of the engine and are not automatically cleared. This may lead to problems such as errors in loading or rendering of resources. There are two solutions:
```